pkgs/development/python-modules/rich-rst/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, docutils
+, rich
+}:
+
+buildPythonPackage rec {
+  pname = "rich-rst";
+  version = "1.1.5";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "wasi-master";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0g1whhw07jgy46a1x85pd23gs356j1cc229wqf5j3x4r11kin6i5";
+  };
+
+  propagatedBuildInputs = [ docutils rich ];
```

Comment:
```suggestion
  propagatedBuildInputs = [ docutils rich ];
  
  # Module has no tests
  noCheck = false;
```

Reply:
Is this meant to say `doCheck`? I can't find any documentation for `noCheck`.

I've pushed some changes that have `doCheck`.

Reply:
yeah, this should be doCheck

Reply:
Sorry, typo.